service-registry/src/main/java/io/servicecomb/serviceregistry/client/http/ServiceRegistryClientImpl.java

```diff
@@ -265,6 +328,51 @@ public boolean registerSchema(String microserviceId, String schemaId, String sch
     return false;
   }
 
+  @Override
+  public boolean syncSchemasAll(Microservice microservice) {
```

Comment:
if there are 600 schemas, will this request too big?

Reply:
so how to deal with such a scene? if local schema is not same with SC's